html-forms/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>
+    Coding Form!
+  </h1>
+  <form action="" method="POST">
+    <div>
+    <label>
```

Comment:
you want to make sure that you labels have a `for` attribute which matches the `id` attribute you want to have on the inputs.